Assets/Scripts/UI/MechaInventoryText.cs

```diff
@@ -0,0 +1,35 @@
+using UnityEngine;
```

Comment:
The UI .cs files also look like they could be condensed into a single UIManager.